holmes/config.py

```diff
@@ -137,7 +137,9 @@ def log_useful_info(self):
                 f"Loaded models: {list(self.llm_model_registry.models.keys())}"
             )
         else:
-            logging.warning("No llm models were loaded")
+            raise Exception(
```

Comment:
This called in both flows after init config is done. Can we consdier move this check there? Is there a reason to keep it in the log_useful_info and not move it to the llm_registry init (in the property?)

Also there are couple of tests failed since until now loading the config were finished succesfully even if no model were loaded. We need to handle those test flows.